specification/eventhub/resource-manager/readme.go.md

```diff
@@ -64,6 +65,15 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2021-06-01-preview/$(namespace)
 ```
 
+### Tag: package-2022-01-preview and go
+
+These settings apply only when `--tag=package-2022-01-preview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-01-preview' && $(go)
```

Comment:
Is the tag `2022-01-preview` or `2022-01-01-preview`? Please double check and resolve the conflict.